tests/ecosystem/upgrade/test_resources.py

```diff
@@ -123,3 +125,30 @@ def test_pod_log_after_upgrade():
     logging.info(
         f"The log '{expected_log_after_upgrade}' appears in all relevant pods."
     )
+
+
+@post_upgrade
+@bugzilla("1973179")
+@skipif_ocs_version("<4.7")
+def test_noobaa_service_mon_after_ocs_upgrade():
+    """
+    test_noobaa_service_mon_after_ocs_upgrade
+
+    Test Procedure:
+    1.Upgrade OCS version [4.6->4.7]
+    2.Check servicemonitors
+    3.Verify 'noobaa-service-monitor' does not exist
+
+    """
+    upgrade_ocs_version = config.UPGRADE.get("upgrade_ocs_version")
+    if Version.coerce(upgrade_ocs_version) >= Version.coerce("4.7.4"):
```

Comment:
From what I remember, we don't use the Z number in ocs-ci. So `.4` won't work or will be ignored I think

Reply:
done